src/plug/stylesheets/base.scss

```diff
@@ -0,0 +1,24 @@
+// Trix editor
+@import 'trix/dist/trix';
+@import 'trix-overrides';
+
+@import 'milligram';
```

Comment:
`Milligram` and `Trix` are imported via npm package now instead of the minified version.

Reply:
That's much better!